modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld

```diff
@@ -2212,6 +2212,12 @@
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<description></description>
+			<name>showDeltasDropDown</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
```

Comment:
Hey @julien, maybe instead of adding this new API to Taglib we just use it internally or do you think it makes sense to keep consistency here with the React component?

For example, we can use the `null` value of `paginationBarDeltas` to set `showDeltasDropDown`.

Reply:
@matuzalemsteles, I'd choose consistency. As you said in https://github.com/liferay/clay/pull/4178 concerning the change for the React component, I don't think you'd expect the dropdown not to be shown because the `paginationBarDeltas` attribute is not passed (in that case you know the tag will use the default value). I think having the same behavior in the tag and the react component makes more sense.

Reply:
Yeah, my concern with the React side is because we had this component for a long time and changing the behavior there would have these problems, as we want to change this behavior but we can't right now, so the suggestion to add a new API but from the side from taglib, we could already go with this behavior. On the React side we will remove this API in an upcoming major version because it is a behavior that makes sense, we only created the API for compatibility.

Probably to change this behavior in the future to taglib we would have to change it in 7.5 or extend this support.

Anyway we'll hold PR until Clay's release tomorrow, once enter, we can send this. Or maybe we wait for the release. cc @nhpatt

Reply:
But it needs a rebase ^

Reply:
Okay, no problem. I'll take care of the rebase too 🙂

Reply:
-- rebased :)